spec/requests/well_known/host_meta_spec.rb

```diff
@@ -24,4 +46,8 @@ def host_meta_xml_template
       </XRD>
     XML
   end
+
+  def host_meta_json_template
```

Comment:
This is fine this way, but always feels a little weird to build string json instead of a hash that gets to_json'd.

Thoughts on switching the examples to something like `expect(response.parsed_body).to ...` and then having this method just be the expected hash/json result?